cli/azd/pkg/apphost/generate_types.go

```diff
@@ -21,6 +21,8 @@ type genKeyVault struct{}
 
 type genContainerApp struct {
 	Image   string
+	Dapr    *genContainerAppManifestTemplateContextDapr
```

Comment:
Without this, the template evaluation logic in my tests was failing because bicep template looks at the Dapr property of the context object, but it was not present.  Looking at the change that introduced this, it wasn't clear there were things to do on the Bicep side of things, but perhaps, @philliphoff understands more.  Would there be a case, @philliphoff where a `container.v0` resource (i.e. an existing container image) would have dapr stuff added to it?  I can image that would be the case - if so, I think we need to make some more changes to ensure that this value gets set correctly.